--- a/fcst/Makefile
+++ b/fcst/Makefile
@@ -35,9 +35,8 @@
 set_var = $(shell { if [ -e "$(1)" ]; then grep -v '^$(2)=' "$(1)"; fi; echo "$(2)=$(3)"; } >/tmp/$(1)-$$$$.tmp && mv /tmp/$(1)-$$$$.tmp $(1))
 
 
-FCSTFLAGS = -DDEBUG_WORK_IN_THREAD -DCONFIG_SCST_TRACING -DCONFIG_SCST_DEBUG \
-	-DCONFIG_SCST_EXTRACHECKS -fno-inline -fno-inline-functions
-FCSTFLAGSRELEASE = -DCONFIG_SCST_TRACING
+FCSTFLAGS = -fno-inline -fno-inline-functions
+FCSTFLAGSRELEASE =
 FCSTFLAGSPERF =
 EXTRA_CFLAGS += -I$(SCST_INC_DIR) $(FCSTFLAGS$(BUILDMODE))
 
--- a/iscsi-scst/kernel/Makefile
+++ b/iscsi-scst/kernel/Makefile
@@ -29,9 +29,9 @@
 	$(call cc-option,-Wno-old-style-declaration) \
 	-Wno-unused-parameter -Wno-missing-field-initializers)
 
-EXTRA_CFLAGS += -DCONFIG_SCST_EXTRACHECKS
+#EXTRA_CFLAGS += -DCONFIG_SCST_EXTRACHECKS
 #EXTRA_CFLAGS += -DCONFIG_SCST_TRACING
-EXTRA_CFLAGS += -DCONFIG_SCST_DEBUG -g -fno-inline -fno-inline-functions
+EXTRA_CFLAGS += -fno-inline -fno-inline-functions
 
 #EXTRA_CFLAGS += -DCONFIG_SCST_ISCSI_DEBUG_DIGEST_FAILURES
 
--- a/qla2x00t/qla2x00-target/Makefile
+++ b/qla2x00t/qla2x00-target/Makefile
@@ -33,9 +33,9 @@
 
 EXTRA_CFLAGS += -W -Wno-unused-parameter -Wno-missing-field-initializers
 
-EXTRA_CFLAGS += -DCONFIG_SCST_EXTRACHECKS 
+#EXTRA_CFLAGS += -DCONFIG_SCST_EXTRACHECKS
 #EXTRA_CFLAGS += -DCONFIG_SCST_TRACING
-EXTRA_CFLAGS += -DCONFIG_SCST_DEBUG -g -fno-inline -fno-inline-functions
+EXTRA_CFLAGS += -fno-inline -fno-inline-functions
 #EXTRA_CFLAGS += -DCONFIG_QLA_TGT_DEBUG_WORK_IN_THREAD
 #EXTRA_CFLAGS += -DCONFIG_QLA_TGT_DEBUG_SRR
 
--- a/scst/src/dev_handlers/Makefile
+++ b/scst/src/dev_handlers/Makefile
@@ -87,10 +87,10 @@
 EXTRA_CFLAGS += -I$(SUBDIRS) -I$(SCST_INC_DIR) $(call enable-Wextra,-Wextra \
 	-Wno-unused-parameter -Wno-missing-field-initializers)
 
-EXTRA_CFLAGS += -DCONFIG_SCST_EXTRACHECKS
+#EXTRA_CFLAGS += -DCONFIG_SCST_EXTRACHECKS
 
 #EXTRA_CFLAGS += -DCONFIG_SCST_TRACING
-EXTRA_CFLAGS += -DCONFIG_SCST_DEBUG -g -fno-inline -fno-inline-functions
+EXTRA_CFLAGS += -fno-inline -fno-inline-functions
 
 clean:
 	rm -f *.o *.ko .*.cmd *.mod.c .*.d .depend Modules.symvers \
--- a/scst_local/Makefile
+++ b/scst_local/Makefile
@@ -14,11 +14,11 @@
 
 #EXTRA_CFLAGS += -DCONFIG_SCST_LOCAL_DIRECT_PROCESSING
 
-EXTRA_CFLAGS += -DCONFIG_SCST_EXTRACHECKS
+#EXTRA_CFLAGS += -DCONFIG_SCST_EXTRACHECKS
 
 #EXTRA_CFLAGS += -DCONFIG_SCST_TRACING
 
-EXTRA_CFLAGS += -DCONFIG_SCST_DEBUG -g -fno-inline -fno-inline-functions
+EXTRA_CFLAGS += -fno-inline -fno-inline-functions
 
 ifneq ($(PATCHLEVEL),)
 obj-m	:= scst_local.o
--- a/srpt/src/Makefile
+++ b/srpt/src/Makefile
@@ -3,9 +3,8 @@
 
 -include $(SUBDIRS)/../build_mode
 
-SRPTFLAGS = -DCONFIG_SCST_TRACING -DCONFIG_SCST_DEBUG \
-	-DCONFIG_SCST_EXTRACHECKS -fno-inline -fno-inline-functions
-SRPTFLAGSRELEASE = -DCONFIG_SCST_TRACING
+SRPTFLAGS = -fno-inline -fno-inline-functions
+SRPTFLAGSRELEASE =
 SRPTFLAGSPERF =
 EXTRA_CFLAGS += $(SRPTFLAGS$(BUILDMODE))
 
